pombola/za_hansard/management/commands/za_hansard_q_and_a_scraper.py

```diff
@@ -209,7 +209,8 @@ def handle_api_question_and_reply(self, data):
                     data['url'])
             return
         house = {
-            'National Assembly': 'N'
+            'National Assembly': 'N',
+            'National Council of Provinces': 'C'
```

Comment:
what in pombola knows what to do with C? is that tested somewhere (either that it handles arbitrary house codes, or that it handles this particular one

Reply:
https://github.com/OpenUpSA/pombola/blob/master/pombola/za_hansard/models.py#L229-L232

Reply:
I searched through all of the occurrences of `.house` in the codebase and there weren't any on question objects.